endoreg_db/views/video/video_correction.py

```
@@ -11,7 +11,7 @@
 
 class VideoCorrectionView(APIView):
```

Pylint (4.0.1)

[refactor] Too few public methods (1/2)

(R0903)